common/ferrostar/src/models.rs

```diff
@@ -171,8 +171,17 @@ pub struct CourseOverGround {
 }
 
 impl CourseOverGround {
-    pub fn new(degrees: u16, accuracy: Option<u16>) -> Self {
-        Self { degrees, accuracy }
+    /// # Arguments
+    ///
+    /// - degrees: The direction in which the user's device is traveling, measured in clockwise degrees from
+    ///            true north (N = 0, E = 90, S = 180, W = 270).
+    /// - accuracy: the accuracy of the course value, measured in degrees.
+    pub fn new(degrees: f64, accuracy: Option<u16>) -> Self {
+        debug_assert!(degrees >= 0.0 && degrees <= 360.0);
```

Comment:
Pretty sure this should be < 360, not <=... https://github.com/georust/geo/pull/1216/files#diff-7119b05faab26ac9655d8487127708ac29437993afa42df72da07de446285d2aR38